apps/a11y-tests/src/tests/__snapshots__/ComponentExamples.test.tsx.snap

```diff
@@ -1645,7 +1645,45 @@ exports[`a11y test checks accessibility of DocumentCard (DocumentCard.Conversati
 
 exports[`a11y test checks accessibility of DocumentCard (DocumentCard.Image.Example) 1`] = `Array []`;
 
-exports[`a11y test checks accessibility of Dropdown (Dropdown.Basic.Example) 1`] = `Array []`;
+exports[`a11y test checks accessibility of Dropdown (Dropdown.Basic.Example) 1`] = `
+Array [
+  Object {
+    "level": "error",
+    "locations": Array [
+      Object {
+        "annotations": Array [
+          Object {
+            "snippet": Object {
+              "text": "<label class=\\"ms-Label ms-Dropdown-label label-26\\" id=\\"Dropdown1-label\\" for=\\"Dropdown1\\">Disabled example with defaultSelectedKey</label>",
+            },
+          },
+        ],
+        "fullyQualifiedLogicalName": "#Dropdown1-label",
+        "physicalLocation": Object {
+          "fileLocation": Object {
+            "uri": "about:blank",
+          },
+        },
+      },
+    ],
+    "message": Object {
+      "richText": "Fix any of the following:
+- Element has insufficient color contrast of 2.63 (foreground color: #a19f9d, background color: #ffffff, font size: 10.5pt, font weight: bold). Expected contrast ratio of 4.5:1",
```

Comment:
Disabled elements on the page do not have to pass the contrast ratio accessibility spec. @betrue-final-final can confirm.